code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -1067,6 +1067,25 @@
 	slowdown = 0.4
 	fire_delay = 6
 
+/obj/item/gun/ballistic/automatic/marksman/sniper/m14
+	name = "m14"
+	desc = "An ancient battle rifle, fitted with a 20 round magazine."
+	icon_prefix = "308"
+	icon_state = "m14rifle"
+	item_state = "rifle"
+	mag_type = /obj/item/ammo_box/magazine/m762/ext
+	extra_damage = 40
```

Comment:
Have you tested this locally by chance? Interested in the time to kill this has on say, combat armor, Legion armor, PA, etc.

Reply:
> Have you tested this locally by chance? Interested in the time to kill this has on say, combat armor, Legion armor, PA, etc.

Exact same time to kill as the m1 garand statistically it’s indifferent. Just has a bigger magazine.

Reply:
Gotcha, sounds good to me then. 
I'd say let's add this gun to the game & then make it a loadout choice for SSgt since we have multiple PRs mucking with said loadouts.

Do you mind if that's what happens?

Reply:
don’t really care what happens with it from now/ post merge

Reply:
Yeah no worries, I'm honestly confused about why this became a big argument.
I'll adjust this PR to not include the loadout change and then put the expectation on Swagmeyer to add it to his loadouts.